File: QE selenium java/Activity5.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {

    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new FirefoxDriver();
        driver.manage().window().maximize();

        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println("Page Title: " + driver.getTitle());

        WebElement checkbox = driver.findElement(By.id("checkbox"));

        System.out.println("Checkbox visible before clicking Remove: "
                + checkbox.isDisplayed());

        WebElement removeButton = driver.findElement(By.id("toggleCheckbox"));
        removeButton.click();

        Thread.sleep(3000);
        try {
            System.out.println("Checkbox visible after clicking Remove: "
                    + checkbox.isDisplayed());
        } catch (Exception e) {
            System.out.println("Checkbox is not visible after clicking Remove");
        }

        driver.quit();
    }
}

File: QE-selenium python/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service
import time

driver = webdriver.Firefox()
driver.maximize_window()

driver.get("https://training-support.net/webelements/dynamic-controls")

print("Page Title:", driver.title)

checkbox = driver.find_element(By.ID, "checkbox")

print("Checkbox visible before clicking Remove:",
      checkbox.is_displayed())

remove_button = driver.find_element(By.ID, "toggleCheckbox")
remove_button.click()

time.sleep(3)

try:
    print("Checkbox visible after clicking Remove:",
          checkbox.is_displayed())
except:
    print("Checkbox is not visible after clicking Remove")

driver.quit()
